apps/condo/domains/common/components/MenuItem.tsx

```diff
@@ -158,7 +158,7 @@ export const MenuItem: React.FC<IMenuItemProps> = (props) => {
         )
 
     const menuItem = (
-        <MenuItemWrapper onClick={handleClick} className={menuItemClassNames} isCollapsed={isCollapsed} {...restWrapperProps}>
+        <MenuItemWrapper id={`index-sidemenu-${path.replace('/', '')}`} onClick={handleClick} className={menuItemClassNames} isCollapsed={isCollapsed} {...restWrapperProps}>
```

Comment:
It's better not to generate ids based on path...Since it can be regex according to 128th line.

It's also not side, but top on mobile.

And what if we wanted to migrate route somehow (for example merge payments and billing sections)...

How about just static id prop?

Reply:
ok